app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -44,6 +46,42 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        if (routeParams == null) {
+            return true;
+        }
+
+        MenuItem item = menu.findItem(R.id.action_edit);
```

Comment:
You should use a more descriptive name than `item`. Also, I'd recommend you add a line below this to also include the call to `SharedPreferences`. Then, you can do:
```java
if (editMenuItem == null || sessionId == null) return true;
```